### ISSUE TITLE
support multiple inputs

### ISSUE THREAD

**Issue by grosser** [coord-e/magicpak#43]
I have a project that needs to use 2 executables, ruby and mysql.
Should I be generating 2 packs or can this be supported ?

**Comment by coord-e** [comment]
I've released v1.4.0 with support for multiple inputs. You can try this like the following:

```
FROM ruby as ruby

ARG ARCH

RUN apt-get update && apt-get install -y mariadb-client

ADD https://github.com/coord-e/magicpak/releases/download/v1.4.0/magicpak-${ARCH}-unknown-linux-musl /usr/bin/magicpak
RUN chmod +x /usr/bin/magicpak
RUN magicpak -v "$(which ruby)" "$(which mysql)" /magicpak
```

**Comment by grosser** [comment]
confirmed working!
